JULIERAJ/KIDS-FIRST-Front
# Feat/122/album page with content

## One Line Description
Create album page with content


## Test Steps
1. Open the application.
2. Navigate to the '/dashboard/albumContent'
3. Ensure that all buttons are fully clickable.

## UI/UX Expected Outcome
-[ [Insert relevant screenshots or videos here]
](https://www.figma.com/proto/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=15054-53609&node-type=frame&t=aYew8hhwa0W1DA6l-0&scaling=scale-down&content-scaling=fixed&page-id=0%3A1&starting-point-node-id=14909%3A60377)

Actual Outcome
![AlbumPageContent](https://github.com/user-attachments/assets/a6e35bab-dc63-4af7-851c-b3333139eca1)

## Checklist
